tests_utils/date_time_utils.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
```

Comment:
This file needs to go in the `utils` directory rather than `tests_utils` directory which is the one for unit tests for utils we write (they are separate from the regular tests directory).  

Can you write some unit tests for these too?  It's basically using pytest directly, and you can look at `tests_axe.py` for some examples but the tests for these will be much simpler, I imagine most of the tests will literally be along the lines of `assert format_date(datetime.date(2024, 2, 1)) = "2024/02/01"` (it may not be exactly that but hopefully gives the gist).